src/utils/beneficial.owner.gov.ts

```diff
@@ -59,6 +62,13 @@ export const getBeneficialOwnerGovById = (req: Request, res: Response, next: Nex
     };
     const appData = getApplicationData(req.session);
 
+    // Redis removal work - Add extra template options if Redis Remove flag is true
```

Comment:
Need to refactor this, once @markpit 's PR is merged. - https://github.com/companieshouse/overseas-entities-web/pull/1143.

Reply:
Done.